media/js/mozorg/home/takeover-2016.js

```diff
@@ -67,18 +70,21 @@
         $options.removeAttr('checked');
     }
 
-    // Shows the take over modal, sets the required entry in
-    // session storage and adds an `esc` key handler to close
-    // the modal.
+    // Shows the take over modal, sets the seven day cookie
+    // and adds an `esc` key handler to close the modal.
     function initTakeOver() {
         // format the currency in browsers that support it
         if (intlNumberFormatSupported) {
             formatCurrency();
         }
-        // show the takeover
-        toggleModal();
-        // set mozorg.takeover.seen to true
-        storeSession();
+
+        // Don't show the takeover if cookies are disabled or not supported
+        if (document.cookie) {
```

Comment:
I wanted to check for cookie support because if a browser doesn't support cookies (or has them blocked/disabled) it would always see the takeover... so no support for cookies, no takeover. The `!Mozilla.Cookie.hasItem('eoy-takeover')` checks for if that specific cookie is absent, but doesn't know if it's absent because the browser doesn't support cookies at all or just hasn't seen the takeover yet.

So I think we do want both checks, but maybe `document.cookie` isn't the best way to do that? It looks like Modernizr is testing for support by just setting a cookie and then detecting it, I'm sure we could rework this slightly to do the same thing. I like the idea of baking that into our cookie helper (heh, baking) but perhaps not in this PR.